src/utils/clmfates_interfaceMod.F90

```diff
@@ -2667,4 +2712,61 @@ subroutine hlm_bounds_to_fates_bounds(hlm, fates)
    
  end subroutine hlm_bounds_to_fates_bounds
 
+ ! ======================================================================================
+
+ subroutine GetAndSetTime()
+
+   ! CLM MODULES
```

Comment:
I'd like a little explanation for what this is doing. It seems to be figuring out the day of the year in order for fates to understand it. I'm a little confused as to it setting the Fates time as I would think the time-stepping determines that.  But, maybe this is just temporarily setting it and then it will be reset later?

Reply:
@ekluzek This is part of reinstating the uninitialized hlm_model_day fix https://github.com/ESCOMP/CTSM/pull/820 that was accidentally lost during the update to release-clm5.0.30 a while back.  The subroutine creation was an effort to modularize the calls to getting and setting time for fates.  The code was repeated three times in the `init`, `dynamic_driv` and `restart` subroutines.  

Reinstating this fix got pushed up the queue as I had to test on Lawrencium while Cheyenne was down and Charlie suggested just rolling the fix into his PR.

Reply:
Oh, I see this was something unrelated that was lost. And it's just work at putting stuff into a subroutine, so it doesn't relate to the rest of the PR, and isn't anything specific for time for the harvesting.

I still like the idea of having a short description of what this is. But, that's the only thing I suggest then.